feature-student/main/src/main/java/com/b1nd/dodam/student/main/navigation/MainNavigation.kt

```diff
@@ -11,8 +11,12 @@ const val MAIN_ROUTE = "main"
 
 fun NavController.navigateToMain(navOptions: NavOptions? = null) = navigate(MAIN_ROUTE, navOptions)
 
-fun NavGraphBuilder.mainScreen(onNightStudyAddClick: () -> Unit) {
+fun NavGraphBuilder.mainScreen(onNightStudyAddClick: () -> Unit, onOutingAddClick: () -> Unit, onSleepOverAddClick: () -> Unit) {
     composable(route = MAIN_ROUTE) {
-        MainScreen(onNightStudyAddClick = onNightStudyAddClick)
+        MainScreen(
+            onNightStudyAddClick = onNightStudyAddClick,
```

Comment:
Please change the param name to navigateToAskNightStudy and also change bottom params like this